npm/deploy/kustomize/overlays/daemon/service.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: npm-deamon-metrics-cluster-service
```

Comment:
per our Prometheus discussion yesterday, we'll need to serve up the same "cluster-metrics" service that we have currently 
https://github.com/Azure/azure-container-networking/blob/6a63ac05a757cdd70c996c149c14ac09a7841458/npm/azure-npm.yaml#L128

Reply:
If we add daemon-specific metrics to npm-metrics-cluster-service, then we'll need to update a scrape config owned by Container Insights team

Reply:
This is a good point. Can we discuss this on chat?